ansible/roles/azure/tasks/create_optional_proxy.yml

```diff
@@ -66,7 +66,7 @@
     name: "{{ azure_proxy_host }}"
     network_interface_names:
       - "{{ azure_proxy_host }}-nic"
-    vm_size: "{{ azure_proxy_host_vm_sku }}"
+    vm_size: Standard_D8s_v3
```

Comment:
Was removing the variable and replacing it with a literal intentional?

Reply:
Yes (in the interim). After this work, I'll add this variable to the `muchos.props` file and make it more generic in a separate PR.